scenarios/25_The_Awakening.cfg

```diff
@@ -268,8 +268,8 @@
         # list here explicitly the bare minimum recruits list player will have after previous scenario;
         # any additional recruits will still be added to it:
         recruit=Peasant,Fencer,Spearman,Bowman,Akladian Clansman,Akladian Wiseman,Akladian Homeguard,Akladian Warrior
-        {GOLD 250 200 150}
-        {INCOME 6 5 4}
+        {GOLD4 250 200 150 100}
```

Comment:
100 is really a very small amount. I wonder if it would be better to make nightmare down to 150 and raise the other difficulties? But again would require playtesting to be sure.

Reply:
Hm... `{260 220 180 140}` perhaps? So that way it's still lower than the old `HARD` amount while not pushing the top end of the scale too high...

Reply:
Okay sure.

Reply:
ok I went with that in [086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).